src/R_display.js

```diff
@@ -0,0 +1,74 @@
+const displayReservationinfo = (movie, receiveReservation, resCounter) => {
+  const displayReservation = (el) => {
+    const ul = document.querySelector('.user-reservation');
+    el.forEach((item) => {
+      const li = document.createElement('li');
+      li.textContent = `${item.date_start} - ${item.date_end} by ${item.username}`;
+      ul.appendChild(li);
+    });
+  };
+
+  const reservationdisp = (el) => {
+    const ul = document.querySelector('.user-reservation');
+    const li = document.createElement('li');
+    li.textContent = `${el.date_start} - ${el.date_end} by ${el.username}`;
+    ul.appendChild(li);
+  };
+
+  const Reservation = document.querySelector('.r-popup');
+  Reservation.addEventListener('click', (event) => {
+    if (event.target.matches('button')) {
+      event.preventDefault();
+      /* eslint-disable */
+      const username = document.querySelector('.input-name').value;
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Done 👍🏼